mautic/mautic
# Update date time text values

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR allows users to enter human readable date values into date fields in the contact fields.

#### Steps to test this PR:
1. Apply PR and try to enter dates with the calendar, and dates with format like NOW, YESTERDAY, TOMORROW, it should not accept values like BLA or any non time word. 
2. these should be converted.
3. Test this in the contact edit page, campaign action to update a contact, form mapped to a datefield, integration mapped to a datefield.
